### DISCUSSION TITLE
Disable file extension check when playing directories introduced in 0.39.0

### DISCUSSION THREAD

**Discussion by spykyvenator** [mpv-player/mpv#15436] [Q&A] [answered]
A lot of my music files have no extension. 
since version 0.39.0 if I try to play a directory with files without extensions: "$ mpv musicdir/", it will exit immediately with an error: Failed to recognize file format.
But if I launch it as "$ mpv musicdir/*" it will work fine.

I am looking to create a patch to restore the old behaviour (for me personally or add a option) but I'm not sure if its mpv and not one of its libraries, I looked in player/loadfile.c but couldn't find where to address the issue.

Where can I change this behaviour?
Thanks in advance!

**Comment by guidocella** [comment, marked as answer]
`--directory-filter-types-clr`

**Reply by spykyvenator** [reply]
Thanks a bunch!
